app/renderer/js/utils/dnd-util.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const ConfigUtil = require(__dirname + '/config-util.js');
+
+function toggle() {
+	const dnd = !ConfigUtil.getConfigItem('dnd', false);
+	const dndSettingList = ['showNotification', 'silent'];
+	if (process.platform === 'win32') {
+		dndSettingList.push('flashTaskbarOnMessage');
```

Comment:
The last thing to be done here is we also need to do this at -  https://github.com/zulip/zulip-electron/blob/master/app/renderer/js/main.js#L95 otherwise the flash feature is broken on windows.